tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -30,30 +35,59 @@ class Settings(Base):
+
     _sound_menu_item_locator = (By.ID, 'menuItem-sound')
-    _keyboard_menu_item_locator = (By.ID, "menuItem-keyboard")
+    _display_menu_item_locator = (By.ID, 'menuItem-display')
+    _homescreen_menu_item_locator = (By.ID, 'menuItem-homescreen')
+    _search_menu_item_locator = (By.ID, 'menuItem-search')
+    _navigation_menu_item_locator = (By.ID, 'menuItem-navigation')
+    _notification_menu_item_locator = (By.ID, 'menuItem-notifications')
+    _date_and_time_menu_item_locator = (By.ID, 'menuItem-dateAndTime')
     _language_menu_item_locator = (By.CSS_SELECTOR, '.menuItem-languageAndRegion')
+    _keyboard_menu_item_locator = (By.ID, "menuItem-keyboard")
+    _theme_menu_item_locator = (By.ID, "menuItem-themes")
+    _addon_menu_item_locator = (By.ID, "menuItem-addons")
+    _achievements_menu_item_locator = (By.ID, "achievements-section")
+
+    _firefox_accounts_menu_item_locator = (By.ID, "menuItem-fxa")
+    _findmydevice_locator = (By.ID, 'menuItem-findmydevice')
+
+    _screen_lock_menu_item_locator = (By.ID, 'menuItem-screenLock')
+    _app_permission_menu_item_locator = (By.ID, 'menuItem-appPermissions')
     _do_not_track_menu_item_locator = (By.ID, 'menuItem-doNotTrack')
+    _browsing_privacy_item_locator = (By.ID, 'menuItem-browsingPrivacy')
+    _privacy_controls_menu_item_locator = (By.ID, 'menuItem-privacyPanel')
     _media_storage_menu_item_locator = (By.CSS_SELECTOR, '.menuItem-mediaStorage')
-    _screen_lock_menu_item_locator = (By.ID, 'menuItem-screenLock')
-    _display_menu_item_locator = (By.ID, 'menuItem-display')
-    _wifi_menu_item_locator = (By.ID, 'menuItem-wifi')
+    _application_storage_menu_item_locator = (By.CSS_SELECTOR, '.menuItem-applicationStorage')
+
     _device_info_menu_item_locator = (By.ID, 'menuItem-deviceInfo')
+    _downloads_menu_item_locator = (By.ID, 'menuItem-downloads')
     _battery_menu_item_locator = (By.CSS_SELECTOR, '.menuItem-battery')
-    _sim_manager_menu_item_locator = (By.ID, 'menuItem-simManager')
-    _date_and_time_menu_item_locator = (By.ID, 'menuItem-dateAndTime')
-    _homescreen_menu_item_locator = (By.ID, 'menuItem-homescreen')
-    _browsing_privacy_item_locator = (By.ID, 'menuItem-browsingPrivacy')
-    _findmydevice_locator = (By.ID, 'menuItem-findmydevice')
+    _accessibility_menu_item_locator = (By.ID, 'menuItem-accessibility')
+    _developer_menu_item_locator = (By.ID, 'menuItem-developer')
+    _improve_menu_item_locator = (By.ID, 'menuItem-improveBrowserOS')
+    _help_menu_item_locator = (By.ID, 'menuItem-help')
+
+    _main_title_locator = (By.ID, 'main-list-title')
 
     def launch(self):
         Base.launch(self)
         Wait(self.marionette).until(
             expected.element_present(*self._app_loaded_locator))
 
+        # this is located at the end of the page.  If this is shown, everything is rendered.
+        Wait(self.marionette).until(
+            expected.element_present(*self._bluetooth_l10n_locator))
+
```

Comment:
If you do this, the previous Wait for *self._app_loaded_locator is not needed anymore, right?